dashboard/test/ui/features/hourOfCodeFinish.feature

```diff
@@ -0,0 +1,21 @@
+Feature: After completing the Hour of Code, the player is directed to a congratulations page
+
+Background:
+  Given I am on "http://studio.code.org/s/mc/reset"
+
+Scenario: Completing Minecraft HoC should go to certificate page and generate a certificate
+  Given I am on "http://studio.code.org/s/mc/stage/1/puzzle/14?noautoplay=true&customSlowMotion=0.1"
+  And I rotate to landscape
+  Then I wait to see a dialog titled "Puzzle 14 of 14"
+  And I close the dialog
+  Then I wait until element "#runButton" is visible
+  Then I wait until the Minecraft game is loaded
+  And I press "runButton"
+  Then I wait to see a congrats dialog with title containing "Keep Playing"
+  And I press the last button with text "Continue"
+  And I get redirected to "/congrats" via "dashboard"
+  And my query params match "\?i\=.*\&s\=bWM\="
+  And there's an image "MC_Hour_Of_Code_Certificate.jpg"
+  And I type "Robo Coder" into "#name"
+  And I press "button:contains(Submit)" using jQuery
+  And I wait to see an image "/api/hour/certificate/"
```

Comment:
Yes, there's an "ignore area" feature in eyes. It's actually pretty easy to use once you figure out which random symbol button to press 

> On Dec 1, 2015, at 9:53, Mehal Shah <notifications@github.com> wrote:
> 
> In dashboard/test/ui/features/hourOfCodeFinish.feature:
> 
> > +Scenario: Completing Minecraft HoC should go to certificate page and generate a certificate
> > +  Given I am on "http://studio.code.org/s/mc/stage/1/puzzle/14?noautoplay=true&customSlowMotion=0.1"
> > +  And I rotate to landscape
> > +  Then I wait to see a dialog titled "Puzzle 14 of 14"
> > +  And I close the dialog
> > +  Then I wait until element "#runButton" is visible
> > +  Then I wait until the Minecraft game is loaded
> > +  And I press "runButton"
> > +  Then I wait to see a congrats dialog with title containing "Keep Playing"
> > +  And I press the last button with text "Continue"
> > +  And I get redirected to "/congrats" via "dashboard"
> > +  And my query params match "\?i\=.*\&s\=bWM\="
> > +  And there's an image "MC_Hour_Of_Code_Certificate.jpg"
> > +  And I type "Robo Coder" into "#name"
> > +  And I press "button:contains(Submit)" using jQuery
> > +  And I wait to see an image "/api/hour/certificate/"
> Might be some challenge with the eyes test because the "sponsored by" can vary. But there should be a way to ignore some section
> 
> —
> Reply to this email directly or view it on GitHub.
>